cpp/src/glm/qn/qn_solvers.cuh

```diff
@@ -49,21 +49,21 @@
 namespace ML {
 namespace GLM {
 
-using MLCommon::alignTo;
+using raft::alignTo;
```

Comment:
since you are doing fully qualified references below, maybe better to remove this line?